### DISCUSSION TITLE
Table 21, Panel 2

### DISCUSSION THREAD

**Discussion by ddmorgan** [cssi-pi-community/2022-meeting#53] [Panel 2 roundtable (Monday afternoon)]
Starting discussion

**Comment by ddmorgan** [comment]
Modest size data sharing is well served (Figshare, zenodo, many projects) and FAIR.  Very large data is huge issue that maybe takes case-by-case approaches.  But data interoperability (e.g., for aggregating for meta-studies) still seems very poor.  Maybe that is a good area for some future focus.

**Comment by ddmorgan** [comment]
There are many groups/projects in specific communities supporting standardized data management.  These are good and NSF should keep supporting these.

**Comment by ddmorgan** [comment]
Consider pushing PIs to use journals that certify codes/scripts and quantifying that at end of grants.

**Comment by ddmorgan** [comment]
Present data archives are serving needs well, although some ongoing NSF to support maintaining, growing new, and innovating in this space is useful.

**Comment by ddmorgan** [comment]
We need more interoperability.  One approach to this is1) Keep the data, 2) Annotate it best you can. 3) Then integrate.  This stage 3 might be a good focus for going forward.

**Comment by ddmorgan** [comment]
NSF should push to have each dataset structured to be interoperable and composable.  E.g., have clear consistent data structures, have APIs, etc.  Then fund the composition of these data sets to form new more valuable integrated data.

**Comment by dtarb** [comment]
I do not want to be too self serving but the HydroShare repository, www.hydroshare.org offers our efforts towards solutions for the Hydrology community that  may have applicability in other fields.  Publications of our work are listed at https://help.hydroshare.org/about-hydroshare/publish/.  I am happy to discuss more - both successes and challenges that relate to the theme of data integrity and provenance.
